src/async_wrap.cc

```diff
@@ -28,6 +28,10 @@
 
 #include "v8.h"
 
+// TODO(mcollina): should this be moved in a more public space?
+// embedders will not be able to use this field
+#define EXECUTION_RESOURCE_FIELD 10
```

Comment:
Please store this on `env->async_hooks()` rather than on the context.

Reply:
Good idea. I've moved it into the AsyncHooks object now.